galsim/chromatic.py

```diff
@@ -924,6 +924,15 @@ def _get_interp_image(self, bandpass, image=None, integrator='trapezoidal', **kw
         # determine combination of self.wave_list and bandpass.wave_list
         wave_list = self._getCombinedWaveList(bandpass)
 
+        if np.min(wave_list) < np.min(self.waves):
+            raise RuntimeError("Requested wavelength %.1f is outside the allowed range:"
+                               " %.1f to %.1f nm"%(np.min(wave_list), np.min(self.waves),
+                                                   np.max(self.waves)))
+        if np.max(wave_list) > np.max(self.waves):
```

Comment:
Any reason not to make this a single ```if``` statement starting with ```if np.min(wave_list) < np.min(self.waves) or np.max(wave_list) > np.max(self.waves):```?  The actual content of the RuntimeError is the same in these two cases, anyway.